tslint.json

```diff
@@ -39,6 +39,19 @@
 		"interface-name": false,
 		"interface-over-type-literal": false,
 		"jsdoc-format": true,
+		"jsx-alignment": true,
+        "jsx-ban-props": [true, ["style", "use className and css"]],
```

Comment:
Indentation is still out of wack (check it on the GitHub diff page if it looks okay on your VS Code)